styx-scheduler-service/src/main/java/com/spotify/styx/BackfillTriggerManager.java

```diff
@@ -123,15 +124,13 @@ private void triggerBackfill(Backfill backfill, Map<String, Long> backfillStates
         // Wait for the trigger execution to complete before proceeding to the next partition
         processed.get();
       } catch (AlreadyInitializedException e) {
-        LOG.warn("tried to trigger backfill for already active state [{}]: {}",
-                 partition, backfill);
-      } catch (InterruptedException e) {
-        Thread.currentThread().interrupt();
-        throw new RuntimeException(e);
+        LOG.warn("tried to trigger backfill for already active state [{}]: {}", partition, backfill);
```

Comment:
Should this even be a warning? I suggest info or lower according to https://github.com/spotify/styx/blob/master/CONTRIBUTING.md ... And in HA (redundant instances) setup I guess we'll have these situations constantly (conflicts when another instance already did something). Would be great to not fill logs with noise because of that.

Reply:
Agree, let's remove this.